srcs/multiboot/mod.rs

```diff
@@ -138,7 +138,7 @@ pub unsafe fn claim_multiboot() {
 							(*mmap_entry).baseaddr as PhysAddr / 4096,
 							(*mmap_entry).length as usize / 4096
 						);
-						bitmap::physmap_as_mut().claim_range(
+						let _ = bitmap::physmap_as_mut().claim_range(
 							(*mmap_entry).baseaddr as PhysAddr,
 							(*mmap_entry).length as usize / 4096
 						);
```

Comment:
Shouldn't we panic if we fail to claim range given from grub?

Reply:
The problem is that the range overlap with the first MiB and cause a panic because it's already mapped, we should change with a new function that doesn't panic at all

Reply:
issued: https://github.com/Harthann/kfs/issues/85